File: 2010_2011_tp_note/Polyaloi.m
function [X] = Polyaloi(nech,n,r,b )
%UNTITLED3 Summary of this function goes here
%   Detailed explanation goes here
X=zeros(nech,1);
for k=1:nech
    X(k)=r/(r+b);
for l = 1:n
    X(k) = (Urne(X(k)*(r+b+l-1),l-1,r,b))/(r+b+l); %Attention aux indices...
end
end

File: 2010_2011_tp_note/Polyaloi.py
import numpy as np

from Urne import Urne


def Polyaloi(nech, n, r, b):
    X = np.zeros(nech)
    for k in range(nech):
        X[k] = r / (r + b)
        for l in range(1, n + 1):
            X[k] = Urne(X[k] * (r + b + l - 1), l - 1, r, b) / (r + b + l)
    return X
